tools/aws_lab_setup/roles/control_node/tasks/main.yml

```diff
@@ -32,31 +32,161 @@
     - control
   when: control_node_install_ansible
 
+- name: Ansible Essentials mode
+  block:
+    - name: Clone lightbulb
+      git:
+        accept_hostkey: yes
+        clone: yes
+        dest: ~{{ username }}/lightbulb
+        repo: https://github.com/ansible/lightbulb.git
+        force: yes
+      become_user: "{{ username }}"
+      tags:
+        - control_node
+        - control
 
-- name: Clone lightbulb
-  git:
-    accept_hostkey: yes
-    clone: yes
-    dest: ~{{ username }}/lightbulb
-    repo: https://github.com/ansible/lightbulb.git
-    force: yes
-  become_user: "{{ username }}"
-  tags:
-    - control_node
-    - control
+    - name: Remove things that students don't need
```

Comment:
This is a bug we should stop repeating. See #109.

Reply:
can this be submitted in a separate PR?  All I did was move this around... I didn't touch the essentials mode stuff except for organizing it, refactoring the manage_ec2_instances role, and providing teardown scripts.

Reply:
If it's a pre-existing bug elsewhere, it makes sense to me if this goes in as-is and we do a separate PR to close instances of #109.